cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java

```diff
@@ -838,6 +897,9 @@ public void testSearchMetrics() throws Exception {
     verifySearchMetricResult("/v3/metrics/search?target=metric&context=namespace.yourspace.adapter.adapter1",
                              ImmutableList.<String>of("system.areads", "system.awrites"));
 
+    verifySearchMetricResult("/v3/metrics/search?target=metric&context=namespace.yourspace.adapter.adapter2",
+                             ImmutableList.<String>of("system.workerreads", "system.workerwrites"));
```

Comment:
Although I see that that's the pattern all over in this file. Wonder why? Intellij warns about this.